src/app/containers/MediaPageMain/index.jsx

```diff
@@ -18,11 +18,17 @@ const renderBlock = ({ script, service }) => block => {
     return null;
   }
 
-  return (
-    <Component script={script} service={service} key={block.text}>
-      {block.text}
-    </Component>
-  );
+  const props = {
+    key: block.text,
+    script,
+    service,
+  };
+
+  if (block.type === 'heading') {
+    props.id = 'content';
```

Comment:
Instead of creating an object and then mutating it you could write an optional object key in the `props` definition e.g.

```js
const props = {
    key: block.text,
    script,
    service,
    ...(block.type === 'heading' && { id: 'content' })
 };
```